app/gui/src/ide/integration.rs

```diff
@@ -71,6 +72,53 @@ impl Model {
             });
         }
     }
+
+    /// Open a project through project manager API.
```

Comment:
"Project manager API" is misleading here. I'd say it is "controller API". But, because the integration is a layer between controllers and view, even this is unnecessary.

Reply:
I changed this docstring and removed `Project manager API` part completely.